server/model/courseSchema.js

```diff
@@ -26,3 +26,913 @@ const courseSchema = new mongoose.Schema({
 const Course = mongoose.model("COURSE", courseSchema);
 
 module.exports = Course;
+
+//Table structure of the schema
```

Comment:
You can remove the comments.

Reply:
I added it so that I can have an understanding of why I used it and what it is doing.